### PR TITLE
add more flow annotations to OrderBook and AutobridgeCalculator classes

### PR DESCRIPTION
**Pull request by darkdarkdragon**

### CODE REVIEW

File: orderbook/src/autobridgecalculator.js [hunk 1]
```diff
@@ -60,7 +68,8 @@ class AutobridgeCalculator {
   }
 
   _calculateInternal(
-    legOnePointer_, legTwoPointer_, offersAutobridged, resolve
+    legOnePointer_: number, legTwoPointer_: number,
+    offersAutobridged: Array<Object>, resolve: (offers: Array<Object>) => void
   ) {
```
Comment (clark800) [inline orderbook/src/autobridgecalculator.js]: Should there be a return type annotation here?

File: orderbook/src/orderbook.js [hunk 2]
```diff
@@ -314,7 +347,7 @@ class OrderBook extends EventEmitter {
       }
     }
 
-    function onLedgerClosedWrapper(message) {
+    function onLedgerClosedWrapper(message: Object) {
```
Comment (clark800) [inline orderbook/src/orderbook.js]: Should there be a return type annotation here?

Comment (darkdarkdragon) [inline orderbook/src/orderbook.js]: I think flow requires only class methods to be strictly annotated.

File: orderbook/src/autobridgecalculator.js (hunk 1 above)
Comment (darkdarkdragon) [inline orderbook/src/autobridgecalculator.js]: Added

**Comment by clark800** [comment]
LGTM

**Pull request merged**